src/app/component/textarea/textarea.js

```diff
@@ -0,0 +1,16 @@
+import './textarea.css';
+import './textarea.media.css';
+import createElement from '../../utils/create-element';
+
+const CssClasses = {
+  TEXTAREA: 'textarea',
+};
+
+function createTextarea() {
+  const textarea = createElement({ tagName: 'textarea', className: CssClasses.TEXTAREA });
+  textarea.rows = 8;
```

Comment:
We shouldn't use such magic numbers.
https://en.wikipedia.org/wiki/Magic_number_(programming)